superset-frontend/package.json

```diff
@@ -80,7 +80,7 @@
     "@emotion/cache": "^11.4.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
-    "@fontsource/inter": "^4.5.7",
+    "@fontsource/inter": "^4.0.0",
```

Comment:
what caused the issue with the previous version of the lib?
Did you try the latest version too?

Reply:
yes sure, it also gives issues with latest version and so I downgrade it into 4.0.0 version from [this PR](https://github.com/apache/superset/pull/19236)